.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '11'
```

Comment:
Ideally, I would like to keep the deploy on JDK 11 as long as possible to make sure that the binaries stay compatible with JDK 11.

Reply:
interesting, is this some requirement from kotlin?